App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Redirect, Switch } from 'react-router-dom';
+import LoginPage from './LoginPage';
+
+const App = () => {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path='/login' component={LoginPage} />
```

Comment:
The application will always redirect to '/login' if the exact path is not '/login'. Consider adding more routes or a 404 page.